recipe-server/client/control_new/components/common/ShieldIdenticon.js

```diff
@@ -3,52 +3,51 @@ import PropTypes from 'prop-types';
 
 import Genome from 'control_new/utils/Genome';
 
-// Solarized colors
-export const colors = [
-  '#002b36', // base03
-  '#073642', // base02
-  '#586e75', // base01
-  '#657b83', // base00
-  '#839496', // base0
-  '#93a1a1', // base1
-  '#eee8d5', // base2
-  '#fdf6e3', // base3
-  '#b58900', // yellow
-  '#cb4b16', // orange
-  '#dc322f', // red
-  '#d33682', // magenta
-  '#6c71c4', // violet
-  '#268bd2', // blue
-  '#2aa198', // cyan
-  '#859900', // green
+
+const emojis = [
+  '😄', '😃', '😀', '😊', '😉', '😍', '😘', '😚', '😗', '😙', '😜', '😝', '😛', '😳', '😁', '😔',
+  '😌', '😒', '😞', '😣', '😢', '😂', '😭', '😪', '😥', '😰', '😅', '😓', '😨', '😱', '😠', '😡',
+  '😤', '😖', '😆', '😋', '😷', '😎', '😴', '😵', '😲', '😟', '😦', '😧', '😈', '👿', '😮', '😬',
+  '😐', '😯', '😶', '😇', '😏', '😑', '👼', '😺', '😻', '😽', '😼', '🙀', '😿', '😹', '😾', '👹',
+  '👺', '🙈', '🙉', '🙊', '💀', '👽', '💩', '🔥', '✨', '🌟', '💫', '💥', '💦', '💧', '💤', '👂',
+  '👀', '👃', '👅', '👄', '👍', '👎', '👌', '👊', '✊', '👋', '✋', '👐', '👆', '🙌', '🙏', '👏',
+  '💪', '💃', '🎩', '👑', '👒', '👟', '👞', '👡', '👠', '👢', '💼', '👜', '👝', '👛', '👓', '🎀',
+  '🌂', '💄', '💛', '💙', '💜', '💚', '💔', '💗', '💓', '💕', '💖', '💞', '💘', '💌', '💋', '💍',
+  '💎', '👣', '🐶', '🐺', '🐱', '🐭', '🐹', '🐰', '🐸', '🐯', '🐨', '🐻', '🐷', '🐽', '🐮', '🐗',
+  '🐵', '🐒', '🐴', '🐑', '🐘', '🐼', '🐧', '🐦', '🐤', '🐥', '🐣', '🐔', '🐍', '🐢', '🐛', '🐝',
+  '🐜', '🐞', '🐌', '🐙', '🐚', '🐠', '🐟', '🐬', '🐳', '🐋', '🐄', '🐏', '🐀', '🐃', '🐅', '🐇',
+  '🐉', '🐎', '🐐', '🐓', '🐕', '🐖', '🐁', '🐂', '🐲', '🐡', '🐊', '🐫', '🐪', '🐆', '🐈', '🐩',
+  '🐾', '💐', '🌸', '🌷', '🍀', '🌹', '🌻', '🌺', '🍁', '🍃', '🍂', '🌿', '🌾', '🍄', '🌵', '🌴',
+  '🌲', '🌳', '🌰', '🌱', '🌼', '🌐', '🌞', '🌝', '🌚', '🌜', '🌛', '🌙', '🌍', '🌎', '🌏', '⭐',
+  '⛅', '⛄', '🌀', '💝', '🎒', '🎓', '🎏', '🎃', '👻', '🎄', '🎁', '🎋', '🎉', '🎈', '🔮', '🎥',
+  '📷', '📹', '📼', '💿', '📀', '💽', '💾', '💻', '📱', '📞', '📟', '📠', '📡', '📺', '📻', '🔊',
+  '🔔', '📢', '⏳', '⏰', '🔓', '🔒', '🔏', '🔐', '🔑', '🔎', '💡', '🔦', '🔆', '🔅', '🔌', '🔋',
+  '🔍', '🛁', '🛀', '🚿', '🚽', '🔧', '🔨', '🚪', '💣', '🔫', '🔪', '💊', '💉', '💰', '💸',
+  '📨', '📬', '📌', '📎', '📕', '📓', '📚', '📖', '🔬', '🔭', '🎨', '🎬', '🎤', '🎵', '🎹', '🎻',
+  '🎺', '🎷', '🎸', '👾', '🎮', '🃏', '🎲', '🎯', '🏈', '🏀', '⚽', '🎾', '🎱', '🏉', '🎳', '⛳',
+  '🚴', '🏁', '🏇', '🏆', '🎿', '🏂', '🏄', '🎣', '🍵', '🍶', '🍼', '🍺', '🍻', '🍸', '🍹', '🍷',
+  '🍴', '🍕', '🍔', '🍟', '🍗', '🍤', '🍞', '🍩', '🍮', '🍦', '🍨', '🍧', '🎂', '🍰', '🍪', '🍫',
+  '🍬', '🍭', '🍯', '🍎', '🍏', '🍊', '🍋', '🍒', '🍇', '🍉', '🍓', '🍑', '🍌', '🍐', '🍍', '🍆',
+  '🍅', '🌽', '🏠', '🏡', '⛵', '🚤', '🚣', '🚀', '🚁', '🚂', '🚎', '🚌', '🚍', '🚙', '🚘', '🚗',
+  '🚕', '🚖', '🚛', '🚚', '🚨', '🚓', '🚔', '🚒', '🚑', '🚐', '🚲', '🚜', '💈', '🚦', '🚧', '🏮',
+  '🎰', '🗿', '🎪', '🎭', '📍', '🚩', '💯',
```

Comment:
> The low-hanging idea I have is to save the resultant SVG to the recipe as plaintext HTML/data URI. I'm not in love with that, though. Maybe we could have it serialize as a basic JS object: { stripes: { count: 2, padding: 0.3, direction: 'horizontal' }, ... }?
>
> The curveball for me is how the Genome.wrapComponent business ties into that approach, any thoughts @mythmon ?

If we want to go this way we could probably ditch `wrapComponent`, and just use Genome to generate a set of props to pass to the component. The question that raises for me is where do we save those serialized props? We can't save them locally, because then different users would have different versions of the shield (or might, if we update things).

Storing stuff like those props in the API doesn't make a lot of sense to me. It is sort of extra noise that I don't think matters to the server. I think I'd like to keep it specific to the front end. Either that, or we should have a generic metadata API for recipes, but those are kind of a pain, and I'm not sure I want to go down that road right now.

Maybe if we ever change this, we could define recipe "generations". These shield identicons are part of generation 1, and if we want to add more features to the identicons, only generation 2 recipes get to use them. We could do it based on recipe creation date or something.

Overall, I think we can leave it as is. The identicons are a minor feature for now, and if we need to improve them in the future, we can deal with that then.